stdlib/map.mli

```diff
@@ -283,7 +283,24 @@ module type S =
     (** Same as {!Map.S.map}, but the function receives as arguments both the
        key and the associated value for each binding of the map. *)
 
+    (** {6 Iterators} *)
 
+    val to_seq : 'a t -> (key * 'a) Seq.t
+    (** Iterate on the whole map, in ascending order
+        @since NEXT_RELEASE *)
+
+    val to_seq_at : key -> 'a t -> (key * 'a) Seq.t
```

Comment:
I'm not very fond of the name.  `to_seq_starting_from` or `to_seq_from`, perhaps?